### ISSUE TITLE
Wishlist: Downloading ONE language by preference

### ISSUE THREAD

**Issue by Kugelblitz360** [Kalanyr/gogrepo#103]
So, would it be possible to have a parameter like "langpref de en" that says: Take the German version and if you can't find one the english version. This would avoid downloading two versions of a game. Or am I missing something in the docs?

Thanks for this excellent script!

**Comment by Kalanyr** [comment]
This seems reasonable, need to work out a way to have this interact well
with the possibility of someone wanting to download multiple languages
though.

**Comment by Sargon-Aelther** [comment]
Semi-related, but I have always wanted something similar for games that are non-English, such as Anstoss 2&3. So in my case I'd prefer English, but have GOGREPOC revert to whatever's available if English is absent, in this case, German would be selected.

**Comment by Kalanyr** [comment]
Yeah, this is a bit of a problem though because what if your preferred
language(s) isn't available but more than 1 fallback is ?  I suppose I can
just set an arbitrary default priority list and let people override it.

But this still becomes a problem if eg people want two.

I need to do some more looking through the current parameter module I'm
using and see if there's some way to do a list of lists as a parameter.
